### ISSUE TITLE
Alignment of signs in settings.js

### ISSUE THREAD

**Issue by dorenberg** [Dashticz/dashticz#488]
Proposal to change lines 804 and further to:

    names: {
      kwh: ' kWh',
      watt: ' Watt',
      gas: ' m3',
      water: ' Liter',
      time: '',

This aligns it with other setting in regards to spacing and official signs.

**Comment by lokonli** [comment]
In the first feedback some people didn't like the spaces.
You can redefine you preferred naming in CONFIG.js. See:
https://dashticz.readthedocs.io/en/beta/blocks/domoticzblocks.html?highlight=units#formatting

**Comment by dorenberg** [comment]
Yes I know, but the standard settings are not used consequent. Sometimes it is "w" and " Watt" or " Watt".
